App/src/pages/dossier/Dossiers.tsx

```diff
@@ -263,7 +266,25 @@ export default function Dossiers() {
                         setCurrentPage={setCurrentPage}
                         currentPage={currentPage}
                         totalResults={totalResults}
+                        useIcons={false}
                     />
+
+                    <span style={{ alignSelf: "flex-end", width: "fit-content", height: "fit-content" }}>
+                        <Button
```

Comment:
In addition to having this button to navigate to the dossiers needing review page, can you also add it to the navbar at the top of the page, since there's already a button there titled "Review Proposal".